prime-angular-frontend/src/app/app-config.module.ts

```diff
@@ -24,6 +24,7 @@ export class AppConfig {
   phoneNumbers: {
     director: string;
   };
+  env: string;
```

Comment:
I would probably make this a union type of `'prod' | 'test' | 'dev'` instead of a string so no one can mistakenly type `production`, `develop`, or just have a typo.